instrumentation/opentelemetry-instrumentation-fastapi/src/opentelemetry/instrumentation/fastapi/__init__.py

```diff
@@ -11,17 +11,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import Optional
 
 import fastapi
 from starlette.routing import Match
 
-from opentelemetry.configuration import Configuration
-from opentelemetry.instrumentation.asgi import OpenTelemetryMiddleware
-from opentelemetry.instrumentation.fastapi.version import __version__  # noqa
 from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.util.http import get_excluded_urls
+from opentelemetry.util.http.asgi import OpenTelemetryMiddleware
```

Comment:
Do we want this ASGI `OpenTelemetryMiddleware` to be part of `util`? I think it's better to have its own package?

Reply:
Maybe, but I am putting any HTTP-related utility code in the same place.